DataFormats/TrackerCommon/src/TrackerTopology.cc

```diff
@@ -396,6 +396,7 @@ SiStripModuleGeometry TrackerTopology::moduleGeometry(const DetId &id) const {
         case 3:
           return SiStripModuleGeometry::W3A;
       }
+      [[fallthrough]];
```

Comment:
I think this should never fall through in real life, though, I would rather break here, after returning an `SiStripModuleGeometry::UNKNOWNGEOMETRY;`